packages/babel-generator/test/fixtures/comments/block-comment-expressions/output.js

```diff
@@ -0,0 +1,9 @@
+fun(/* a */ a);
+
+function fun(/* a */ a, /* b */ b) {
+  return /* c */ c;
+}
+/** d */
+
+
```

Comment:
Too many newlines? :thinking: IIRC, we had some logic to avoid pushing more than two `\n`s

Reply:
yeah that looks ugly. Let me see if this is a regression and fix it if it is.